pkgs/top-level/python-packages.nix

```diff
@@ -1900,6 +1900,8 @@ in {
 
   eth-utils = callPackage ../development/python-modules/eth-utils { };
 
+  impacket = callPackage ../development/python-modules/impacket { };
```

Comment:
Is this required for `sensu-check-wmiplus`?

Reply:
Leftover from py-wmi-client, ref #69336

I'll clean this up.